pages/linux/pacman.md

```diff
@@ -5,32 +5,32 @@
 
 - Synchronize and update all packages:
 
-`pacman -Syu`
+`pacman --sync --refresh --sysupgrade`
 
 - Install a new package:
 
-`pacman -S {{package_name}}`
+`pacman --sync {{package_name}}`
 
 - Remove a package and its dependencies:
 
-`pacman -Rs {{package_name}}`
+`pacman --remove --recursive {{package_name}}`
 
 - Search the package database for a regular expression or keyword:
 
-`pacman -Ss "{{search_pattern}}"`
+`pacman --sync --search "{{search_pattern}}"`
 
 - List installed packages and versions:
 
-`pacman -Q`
+`pacman --query`
 
 - List only the explicitly installed packages and versions:
 
-`pacman -Qe`
+`pacman --query --explicit`
 
-- Find which package owns a certain file:
+- List orphan packages (installed as dependencies but not required by any package):
 
-`pacman -Qo {{filename}}`
+`pacman --query --unrequired --deps --quiet`
 
 - Empty package cache to free up space:
 
-`pacman -Scc`
+`pacman --sync cc`
```

Comment:
```suggestion
`pacman --sync --clean`
```

Reply:
Actually that should be `--clean --clean`

Reply:
Ps this was left as a placeholder!

By no means did I intend to leave it as edited.

It was a mix of a placeholder and me just going to sleep since it was very late and just wanting to commit

Reply:
```suggestion
`pacman --sync --clean --clean`
```

Once we have received a reply from @navarroaxel we will commit the suggestion. I think it needs to be clean clean however he might have other opinions on what is correct.

Reply:
double `clean` is ok for me.

Reply:
but with this example description, 1 `clean` sounds better. Because you are saying, use 2 for ....

Reply:
1 clean signifies just `pacman -Sc` which isn't the same

https://wiki.archlinux.org/title/Pacman#Cleaningthepackagecache